modules/op-event-pruner-configuration.adoc

```diff
@@ -0,0 +1,98 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="event-pruner-configuration_{context}"]
+= Enabling the event-based pruner
+
+:FeatureName: The event-based pruner
+include::snippets/technology-preview.adoc[]
+
+You can use the event-based `tektonpruner` controller to automatically delete completed resources, such as `PipelineRuns` and `TaskRuns`, based on configurable policies. Unlike the default job-based pruner, the event-based pruner listens for resource events and prunes resources in near real time.
+
+You can use the event-based pruner alongside:
+
+* Resource annotations
+* {tekton-results}
+* {PAC}
+
+[NOTE]
+====
+The Operator does not currently restrict the use of these features with the event-based pruner. However, future releases might introduce validation if conflicts are observed.
```

Comment:
The Operator does not restrict the use of these features with the event-based pruner. However, future releases might introduce validation if conflicts are detected.